source/_integrations/egps.markdown

```diff
@@ -0,0 +1,38 @@
+---
+title: Energenie Power-Strips (EGPS)
+description: Instructions on how to integrate Energenie Power-Strip switches into Home Assistant.
+ha_category:
+  - Switch
+ha_iot_class: Local Polling
+ha_release: '2024.4'
+ha_codeowners:
+  - '@gnumpi'
+ha_domain: egps
+ha_platforms:
+  - switch
+ha_integration_type: integration
+---
+
+This integration allows you to include [Energenie USB Power-Sockets](https://energenie.com/item.aspx?id=7556&lang=de) into your Home Assistant setup.
+
+{% include integrations/config_flow.md %}
+
+## Troubleshooting
+
+Depending on your system configuration, it may be necessary to grant explicit user access rights to the USB device by creating a udev rule.
+
+1. find the *vendor_id* and *product_id* of the USB device by calling `lsusb`:
+
+```bash
+lsusb
+
+#e.g.: Bus 001 Device 005: ID 04b4:fd15 Cypress Semiconductor Corp. Energenie EG-PMS2
+```
+
+2. create an udev rule by calling:
+
+```bash
+sudo echo 'SUBSYSTEM=="usb", ATTR{idVendor}=="04b4", ATTR{idProduct}=="fd15", MODE="0666"' > /lib/udev/rules.d/60-energenie-usb.rules
+sudo udevadm control --reload-rules
+sudo udevadm trigger
+```
```

Comment:
This doesn't seem to apply to the larger sum of our user base?

As in, this probably won't work like this on the Home Assistant Operating System

Reply:
Finally I found the time to setup a raspberry with Home Assistant Operating System. Fortunately, the 'egps' integration works right away, there is no need in setting specific user rights. Do you recommend removing the *Troubleshooting* section completely then? I could also be more specific in saying that it helps if running the container or core version of HA.